src/transformers/models/mask2former/modeling_mask2former.py

```diff
@@ -2224,6 +2391,7 @@ def forward(
         self,
         pixel_values: Tensor,
         pixel_mask: Optional[Tensor] = None,
+        video_input: Optional[bool] = False,
```

Comment:
Adding new arguments to the forward functions of existing models create backward compatibility issues, could you add this as a configuration parameter instead?
```suggestion
```